### PR TITLE
Minor fixes and started work on graph analyser

### PR DESCRIPTION
**Pull request by danielchriscarter**

### CODE REVIEW

File: tools/scraper/manual-data/attributes/versions.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "6.0 and below, 5.1 and below, 6.0 and below, 6.0": "([1-5]\\.[0-9]\\.[0-9])|(6\\.0\\.[0-9])",
+  "6.0 and below": "([1-5]\\.[0-9]\\.[0-9])|(6\\.0\\.[0-9])",
+  "5.1 and below": "([1-4]\\.[0-9]\\.[0-9])|(5\\.[0-1]\\.[0-9])",
+  "5.1 and below, 5.0 and 5.1": "([1-4]\\.[0-9]\\.[0-9])|(5\\.[0-1]\\.[0-9])",
+  " 5.1 and below": "([1-4]\\.[0-9]\\.[0-9])|(5\\.[0-1]\\.[0-9])",
```
Comment (drt24) [inline tools/scraper/manual-data/attributes/versions.json:6]: Leading space.

File: tools/scraper/wiper.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    return data
+
+def write_manual_data(cve, data):
+    """Writes manually entered data out to a file"""
+    with open('manual-data/{cve}'.format(cve=cve), 'w') as f:
+        json.dump(data, f, indent=2)
+
+
+print('This program will erase data fields from the manual data files. Please use with care')
+key = input('Please enter the field to erase: ')
```
Comment (drt24) [inline tools/scraper/wiper.py:23]: `key` isn't actually used later in the file.

**Review by drt24 — APPROVED** [review]
A couple of minor comments to fix later.

File: tools/scraper/manual-data/attributes/versions.json (hunk 1 above)
Comment (danielchriscarter) [reply tools/scraper/manual-data/attributes/versions.json:6]: That's there in the source data, but should hopefully be stripping this off and so working without it now

File: tools/scraper/wiper.py (hunk 2 above)
Comment (danielchriscarter) [reply tools/scraper/wiper.py:23]: Oops - now fixed

**Pull request merged**